packages/agentive-kit/src/agentive_kit/door/data/checks/checks-python.sh

```
@@ -0,0 +1,205 @@
+fi
+echo
+
+# 5. Pattern lint (project-specific DK rules)
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "5/7 🔍 Running pattern lint (DK rules)..."
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+PY_FILES=$(find scripts/ tests/ -name '*.py' 2>/dev/null)
+if [ ! -f "$CORE_DIR/pattern_lint.py" ]; then
+    echo "⚠️  pattern_lint.py not present (packaged repo) — step skipped"
+elif [ -n "$PY_FILES" ]; then
+    if python3 "$CORE_DIR/pattern_lint.py" $PY_FILES 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)